### ISSUE TITLE
HTML Entities in RightMangement 

### ISSUE THREAD

**Issue by katgirl** [contao/core#8002]
Create a new BackendModule ```config.php```

```php
array_insert($GLOBALS['BE_MOD'], 1, array('kirsten.roschanski' => array
(
	'test' => array
	(
		'tables'      => array('tl_mbym_resources'),
		'icon'        => 'system/modules/resource_database/assets/resources.gif'
	)
)));
```

Create a language file ```modules.php```
```php
$GLOBALS['TL_LANG']['MOD']['kirsten.roschanski'] = array(
  'kirsten<span style="color:#f06">.</span>roschanski',
  ''
);

$GLOBALS['TL_LANG']['MOD']['test'] = array(
  'Test',
  ''
);
```

You can't select it in RightManagment.

![image](https://cloud.githubusercontent.com/assets/1223243/9755674/304ddfcc-56d4-11e5-84ed-0a2c7fed683c.png)

```html
<div class="checkbox_toggler"><a href="contao/main.php?do=group&amp;act=edit&amp;id=1&amp;rt=988d81f45a18ec08542f91dd4232d949&amp;cbc=cbc_modules_kirstenspan-stylecolorf06-spanroschanski&amp;ref=16826246" onclick="AjaxRequest.toggleCheckboxGroup(this,'cbc_modules_kirstenspan-stylecolorf06-spanroschanski');Backend.getScrollOffset();return false"><img src="system/themes/mbym/images/folMinus.gif" width="18" height="18" alt="toggle checkbox group"></a>kirsten<span style="color:#f06">.</span>roschanski</div>

<fieldset id="cbc_modules_kirstenspan-stylecolorf06-spanroschanski" class="tl_checkbox_container checkbox_options" style="display:block"><input type="checkbox" id="check_all_cbc_modules_kirstenspan-stylecolorf06-spanroschanski" class="tl_checkbox" onclick="Backend.toggleCheckboxGroup(this, 'cbc_modules_kirstenspan-stylecolorf06-spanroschanski')"> <label for="check_all_cbc_modules_kirstenspan-stylecolorf06-spanroschanski" style="color:#a6a6a6"><em>Alle auswählen</em></label><br><input type="checkbox" name="modules[]" id="opt_modules_kirsten<span style=" color:#f06"="">.roschanski_0" class="tl_checkbox" value="test" onfocus="Backend.getScrollOffset()"&gt; <label for="opt_modules_kirsten<span style=" color:#f06"="">.roschanski_0"&gt;Test</label></fieldset>
```

**Comment by leofeyer** [comment]
?

**Comment by leofeyer** [comment]
Fixed in abca821eca06d3c2d027eaa61337b6bb10cf79dc.
